include/sampleflow/consumers/last_sample.h

```diff
@@ -13,19 +13,30 @@
 //
 // ---------------------------------------------------------------------
 
-#ifndef SAMPLEFLOW_CONSUMERS_LAST_SAMPLE_H
-#define SAMPLEFLOW_CONSUMERS_LAST_SAMPLE_H
+#ifndef SAMPLEFLOW_CONSUMERS_HISTOGRAM_H
+#define SAMPLEFLOW_CONSUMERS_HISTOGRAM_H
```

Comment:
I think you made a mistake in this file. This file should implement the `LastSample` consumer, but I believe that you overwrote it with the contents of the `histogram.h` file.

Reply:
In the next pull request, it would be fixed.